packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -38,14 +38,17 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu }) => {
             setDialogStatus(true);
         }
     };
+
     React.useEffect(() => {
         document.addEventListener('click', onClickExternalLink);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
pls help to set off on .eslintrc for this lint type

Reply:
@mustofa-binary it's nice to have as a sanity check? to make sure your effects aren't missing any dependencies as this could lead to some sneaky bugs.

Reply:
i see, you're right, i was just afraid that we will used to add eslint-disable-next-line to most of our code, i guess warning is fine for this 👍